internal/ovsexporter/ovsexporter.go

```diff
@@ -18,20 +20,41 @@ const (
 
 // A collector aggregates Open vSwitch Prometheus collectors.
 type collector struct {
-	mu sync.Mutex
-	cs []prometheus.Collector
+	mu               sync.Mutex
+	cs               []prometheus.Collector
+	conntrackEnabled bool
```

Comment:
is there ever a situation where this is `false` but `aggregator` is non-`nil`?
i don't believe there would be , in which case this extra bool doesn't provide any value